retail-managment-system/rms-ui/src/styles.scss

```diff
@@ -1,3 +1,20 @@
+
+@use '@angular/material' as mat;
+@include mat.core();
+
+$rms-ui-primary: mat.define-palette(mat.$indigo-palette);
+$rms-ui-accent: mat.define-palette(mat.$pink-palette, A200, A100, A400);
+
+$rms-ui-warn: mat.define-palette(mat.$red-palette);
+
+$rms-ui-theme: mat.define-light-theme((
+  color: (
+    primary: $rms-ui-primary,
+    accent: $rms-ui-accent,
+    warn: $rms-ui-warn,
+  )
+));
+@include mat.all-component-themes($rms-ui-theme);
```

Comment:
why add this lines from line 1 to 17?

Reply:
Using it to style the table with angular material

Reply:
without it style of table not work?